Sections/Dashboard.cs

```diff
@@ -1,18 +1,25 @@
+using System.Collections.ObjectModel;
+using SimHub.Plugins.OutputPlugins.GraphicalDash;
+
 namespace DahlDesign.Plugin.Categories
 {
     public class Dashboard
     {
         private readonly DahlDesign Base;
 
- 
+        public Screen LeftScreen;
+        public Screen RightScreen;
         public Dashboard(DahlDesign dahlDesign)
         {
             Base = dahlDesign;
 
+            LeftScreen = new Screen(new string[] { "TimeAttack", "Sector", "LapTime", "Quali", "Race", "Race2", "Race3", "ShiftPoint", "Map" });
```

Comment:
We should change the name of these pages:
- Time1
- Time2
- Time3
- Qualy
- Race1
- Race2
- Race3
- Practice1
- Practice2